### PR TITLE
chore(content/update): Add Alpine 3.18 Update to Docs

### PR DESCRIPTION
**Pull request by psavidis**
relates to:  camunda/camunda-bpm-platform#3234

### CODE REVIEW

**Review by mboskamp — COMMENTED** [review]
Hey @psavidis,
In my opinion, we can improve the wording a bit here and there. I left comments where I think we can clarify things.
What do you think? This would have to be adjusted on the backport branches as well.
The headers on the other branches look good. 👍

File: content/update/minor/719-to-720/_index.md [hunk 1]
```diff
@@ -16,18 +16,19 @@ menu:
...
+5. For administrators: [Explicit JUEL module on Jakarta Expression Language 4](#explicit-juel-module-on-jakarta-expression-language-4)
+6. For developers: [JavaScript external task client re-throws errors on task service APIs](#javascript-external-task-client-rethrows-errors-on-task-service-apis)
+7. For developers: [Explicit asset declaration in Java web app plugins](#explicit-asset-declaration-in-java-web-app-plugins)
+8. For developers: [Discontinue support for handling JPA entities as variables](#discontinue-support-for-handling-jpa-entities-as-variables)
+9. For developers: [Quarkus 3 update](#quarkus-3-update)
+10. For developers: [Spring Framework 6.0 support](#spring-framework-6-0-support)
+11. For developers: [Upgrade to Spring Boot 3.1](#upgrade-to-spring-boot-3-1)
+    * For developers: [External Task Client Spring Boot Starter requires JDK 17](#external-task-client-spring-boot-starter-requires-jdk-17)
+12. For developers: [Camunda Platform Run requires JDK 17](#camunda-platform-run-requires-jdk-17)
+13. For developers: [Update Alpine Base Docker Image to 3.18](#update-to-alpine-3-18)
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/minor/719-to-720/_index.md]: Applied suggestion

File: content/update/minor/719-to-720/_index.md [hunk 2]
```diff
@@ -212,3 +213,9 @@ After adopting Spring Boot 3, the External Task Client Spring Boot Starter requi
 
 Starting with Camunda Platform 7.20, the Camunda Platform Run distribution requires Java Runtime Environment 17 installed.
 
+# Update to Alpine 3.18
+
+This release comes with an update of the Alpine base docker image. See the detailed list of changes in the official [release notes][alpine318] page.
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/minor/719-to-720/_index.md]: Applied suggestion

File: content/update/patch-level.md [hunk 3]
```diff
@@ -444,6 +444,14 @@ Requests for undeclared assets will be rejected, and it will likely render your
 [custom-script]: {{< ref "/webapps/cockpit/extend/configuration#custom-scripts" >}}
 [frontend-modules]: {{< ref "/webapps/cockpit/extend/plugins#structure-of-a-frontend-module" >}}
 
+## 7.19.4 to 7.19.5 / 7.18.9 to 7.18.10 / 7.17.14 to 7.17.15
+
+### Update to Alpine 3.18
+
+This release comes with an update of the Alpine base docker image. See the detailed list of changes in the official [release notes][alpine318] page.
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/patch-level.md:451]: Applied suggestion

File: content/update/patch-level.md [hunk 4]
```diff
@@ -444,6 +444,14 @@ Requests for undeclared assets will be rejected, and it will likely render your
 [custom-script]: {{< ref "/webapps/cockpit/extend/configuration#custom-scripts" >}}
 [frontend-modules]: {{< ref "/webapps/cockpit/extend/plugins#structure-of-a-frontend-module" >}}
 
+## 7.19.4 to 7.19.5 / 7.18.9 to 7.18.10 / 7.17.14 to 7.17.15
+
+### Update to Alpine 3.18
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/patch-level.md]: Applied suggestion

File: content/update/patch-level.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (psavidis) [reply content/update/patch-level.md]: No, i ommited during another commit suggestion & working on the branch to fix the link to be navigable. Now its applied.

**Pull request merged**